examples/package.json

```diff
@@ -10,9 +10,9 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "^31.0.2",
-    "react": "16.5.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz",
+    "expo": "^33.0.0",
```

Comment:
Should it be expo version 34 to compatible with expo 34?

Reply:
I chose to use 33, since at that time, the Google 64bit requirement was handled, and in that version, you would only get warnings for such changes and I have made in this PR. So if there are anyone else who is building an app with a lot of libraries, they are maybe forced to stay on SDK 33, until all their dependencies have made the needed changes to be compatible with SDK 34. 

If you want, it could easily be updated to 34 :)

Reply:
Okay, thank you for the explanation :) I have not been working with Expo for so long, therefore I did not follow up Expo changes. Thank you, I'm gonna merge and release a new version :) Thank you @AndersGerner for your contribution.